nightly/nightly.txt

```diff
@@ -22,6 +22,7 @@ pytest sanity/rpc_tx_forwarding.py
 pytest --timeout=240 sanity/skip_epoch.py
 pytest --timeout=240 sanity/one_val.py
 pytest --timeout=240 sanity/lightclnt.py
+pytest sanity/rpc_light_client_execution_outcome_proof.py
```

Comment:
Isn't it sane to have some sort of timeout?

Reply:
default is 180s, which I think is enough